nipy/algorithms/utils/tests/test_matrices.py

```diff
@@ -31,7 +31,7 @@ def test_matrix_rank():
     eps = np.finfo(X.dtype).eps
     assert_equal(matrix_rank(X, tol=0), 10)
     assert_equal(matrix_rank(X, tol=S.min() - eps), 10)
-    assert_equal(matrix_rank(X, tol=S.min() + eps), 9)
+    assert_equal(matrix_rank(X, tol=S.min() + eps), 10)
```

Comment:
On my laptop and a couple of linux boxes I have access too, the edited version of the test fails, as I would expect - so I am not sure what's happening with the travis-ci test.